packages/toolkit/src/lib/vdp-sdk/pipeline/queries.ts

```diff
@@ -117,7 +117,7 @@ export async function listUserPipelinesQuery(
     const pipelines: Pipeline[] = [];
 
     const queryString = getQueryString({
-      baseURL: `${userName}/pipelines?view=VIEW_FULL`,
+      baseURL: `/users/${userName}/pipelines?view=VIEW_FULL`,
```

Comment:
@iamnamananand996 Because we are not sure about whether the pipeline is for user or organization and right now we all use "entityName" as this combination of identity. 

Could you leave this as it but change the names to entityName?

So the queryString will be `${entityName}/pipelines?view=VIEW_FULL`